README.md

```diff
@@ -74,11 +74,12 @@ requirements that this script has that may not already be in place:
 * Socat        - Allows the script to communicate with the Minecraft server.
 * Iptables     - Although not explicitly required, a good firewall should be
                  installed.
+* Build-essential - Allows the script to run the makefile for configuration.
```

Comment:
GNU Make is what we actually require.  The build-essential package also pulls in gcc, for which we have no need.